src/styles/dreamweaver-light.css

```diff
@@ -0,0 +1,85 @@
+/* inspired by Dreamweaver code colors, created by https://www.TNTcode.com */
+
+.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #fff;
+  font-family: "Courier New", monospace;
+  font-size: 13px;
```

Comment:
Don't embellish please, no font family or sizing (keep it the same as all the other themes).